doc/api/module.md

```diff
@@ -113,7 +113,9 @@ changes:
     `initialize` hook.
 
 Register a module that exports [hooks][] that customize Node.js module
-resolution and loading behavior. See [Customization hooks][].
+resolution and loading behavior. Registering modules that export [hooks][] only
+has an effect on the main thread. All worker threads inherit the customization
+hooks chain of the main thread. See [Customization hooks][].
```

Comment:
“only has an effect on the main thread” sounds like the hooks don’t apply to worker threads. I just wouldn’t add anything here. I think the default assumption is that customization hooks apply everywhere, and thanks to the recent PR, now they do.

Reply:
ok, removed the change in `cli.md` now.